src/NzbDrone.Core/DecisionEngine/Specifications/ReleaseRestrictionsSpecification.cs

```diff
@@ -30,6 +30,13 @@ public virtual Decision IsSatisfiedBy(RemoteEpisode subject, SearchCriteriaBase
             _logger.Debug("Checking if release meets restrictions: {0}", subject);
 
             var title = subject.Release.Title;
+            var origin = subject.Release.Origin ?? "";
+            var source = subject.Release.Source ?? "";
+            var container = subject.Release.Container ?? "";
+            var codec = subject.Release.Codec ?? "";
+
+            title += origin + source + container + codec;           var title = subject.Release.Title;
```

Comment:
`title` is already instantiated here (and you don't want two "lines" on one line).

I also don't think we'd want to combine all those variables into a single string to compare, if you for example you had a regex restriction that was `\bx265\b` it'd fail because you'd end up without any spaces/separation between the words. Checking each field individually would be more reliable.

Reply:
I agree on the regex I was just thinking about that. I think a simple delimiter like a “.” would work though to prevent issues